projectroles/management/commands/cleanappsettings.py

```diff
@@ -42,11 +44,27 @@ def handle(self, *args, **options):
             else:
                 def_kwargs['app_name'] = 'projectroles'
             try:
-                app_settings.get_definition(**def_kwargs)
+                definition = app_settings.get_definition(**def_kwargs)
             except ValueError:
                 logger.info(
-                    'Deleting "{}" from project "{}"'.format(
-                        get_setting_str(s), s.project.title
+                    'Deleting "{}" from project "{}": {}'.format(
```

Comment:
Since there is repetition for "Deleting x from project y", I recommend making this prefix a constant with the appropriate `{placeholders}` so we can simply refer and reformat to that, both here and in the test cases.